guides-markdown/connect-local-nodes.md

```diff
@@ -0,0 +1,153 @@
+# Connect two local nodes
+
+```post-author
+Orfeas Stefanos Thyfronitis Litos
+```
+
+```post-description
+Start and manually connect two regtest nodes in a single script.
+```
+
+When we start two nodes in the regtest (loopback network, 127.0.0.1) with
+manually set ports, they don't get connected automatically because they don't
+know each other's port. We thus have to connect them manually. In this example
+we will fire up one SPV and one Full node and initiate a connection from the
+SPV side. The SPV node is given the address of the Full node and attempts to
+connect. If everything goes well, the two nodes connect. Finally they are
+gracefully closed.
+
+The script should work out of the box, the only requirement is having `bcoin`
+installed.
```

Comment:
I had to install `p-event` on my system as well :-) 
Can you add a little instruction for that? Maybe a one-line description of the package as well?

Reply:
Oops, I forgot to change the prose as well.